src/bindings/html/dom.js

```diff
@@ -82,6 +88,11 @@ function camelCaseToDashed(string) {
 }
 
 export function translateElement(element) {
+  if (!this.service.languages) {
```

Comment:
agh, and the problem I hit here is that in non-pretranslated case, translateDocument is fired before service.languages is set :/